src/inmanta/data/__init__.py

```diff
@@ -46,25 +46,19 @@
 from asyncpg.protocol import Record
 
 import inmanta.db.versions
+import inmanta.protocol
+import inmanta.types
```

Comment:
1. Why do we need protocol in here ?
2. I think we can remove `import inmanta.types` if we import `ResourceType` from `inmanta.types` + change where we use it (only 1 place I think)